dev-scripts/quick

```diff
@@ -48,6 +48,7 @@ if [ -z $TARGET ] || [ $TARGET = "server" ]; then
     --tag "${REPO}"/rancher:"${TAG}" \
     --platform="${OS}/${ARCH}" \
     --target server \
+    --push \
```

Comment:
What does this do in this context?

Reply:
Second this question.

Reply:
It's a cheap workaround to get `make quick` to work. Without it something doesn't line up right with the docker image tags and I get a build error, because nothing in the build cache contains the correct tags. With this docker images with the correct tags are pushed to the registry, so it's not a problem that they aren't in the local build cache.
I've accidentally left it in when pushing, it's not connected to the bug.